src/Makefile

```diff
@@ -13,6 +13,7 @@ else
 	SHOBJ_CFLAGS ?= -dynamic -fno-common -g -ggdb
 	SHOBJ_LDFLAGS ?= -bundle -undefined dynamic_lookup
 endif
+
```

Comment:
lines 10+11
```c
# We add -Werror to CFLAGS only on Linux since on OSX there are still tons of warnings
CFLAGS +=-Werror  
```
for linux only  should be removed if -Werror is now global